docs/index.bs

```diff
@@ -129,8 +129,10 @@ spec: csp2; urlPrefix: https://w3c.github.io/webappsec-csp/2/
 
 spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
     type: dfn
+        text: append to header list; url: concept-header-list-append
         text: basic filtered response; url: concept-filtered-response-basic
         text: cancel a ReadableStream; url: concept-cancel-readablestream
+        text: cloning the request; url: concept-request-clone
```

Comment:
I get the feeling this is the wrong way to do this, but I'm not sure what the right way is